unison-core/src/Unison/Term.hs

```diff
@@ -474,7 +474,7 @@ var = ABT.annotatedVar
 var' :: Var v => Text -> Term0' vt v
 var' = var() . Var.named
 
-ref :: Ord v => a -> Reference -> Term2 vt at ap v a
+ref :: forall v a vt at ap. Ord v => a -> Reference -> Term2 vt at ap v a
```

Comment:
Any particular reason for this one? I don't think it's helping.

Reply:
We wanted to pass `@t` to this function for the type parameter `v` but found that it was getting applied to the parameter `a`. Perhaps no longer relevant after some simplifications elsewhere.